extensions/cornerstone/src/Viewport/OHIFCornerstoneViewport.tsx

```diff
@@ -345,6 +350,27 @@ const OHIFCornerstoneViewport = React.memo(props => {
     };
   }, [displaySets, elementRef, viewportId]);
 
+  // Set up the window level action menu in the viewport action corners.
+  useEffect(() => {
+    const wlActionMenu = getWindowLevelActionMenu({
+      viewportId,
+      viewportElem: elementRef.current,
+      displaySets,
+      servicesManager,
+      commandsManager,
+      verticalDirection: AllInOneMenu.VerticalDirection.BottomToTop,
+      horizontalDirection: AllInOneMenu.HorizontalDirection.LeftToRight,
+    });
+
+    viewportActionCornersService.setActionComponent({
```

Comment:
Please consider adding a to-do item suggesting the use of the customizationService to place components and customize their placements.